### PR TITLE
Add Dynamoose request schema

### PR DESCRIPTION
**Pull request by DomBelcher**
This adds a dynamoose schema for Alma Request data.

### CODE REVIEW

File: src/request-schema.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  request_type: String,
+  request_sub_type: {
+    value: String,
+    desc: String
+  },
+  request_status: String,
+  pickup_location: String,
+  pickup_location_type: String,
+  pickup_location_library: String,
+  material_type: {
```
Comment (stephenrob) [inline src/request-schema.js:28]: Can you explain what this formatting does?

File: src/request-schema.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+const Schema = dynamoose.Schema
+
+const requestSchema = new Schema({
+  request_id: {
+    type: String,
+    hashKey: true
+  },
+  user_primary_id: String,
+  request_type: String,
+  request_sub_type: {
```
Comment (stephenrob) [inline src/request-schema.js:17]: Can you explain what this formatting does?

**Review by stephenrob — COMMENTED** [review]
Couple of bits i'm interested in knowing more about as i haven't used the ORM library before.

File: src/request-schema.js (hunk 2 above)
Comment (DomBelcher) [reply src/request-schema.js:17]: The intention was to define an object with the props 'value' and 'desc' on it, as they are in Alma. I've tested this and this isn't the correct way to do this in dynamoose, I'll update it now

**Review by stephenrob — DISMISSED** [review]

**Review by stephenrob — APPROVED** [review]

**Pull request merged**